src/ResourceManager/EventGrid/Commands.EventGrid.Test/Commands.EventGrid.Test.csproj

```diff
@@ -40,15 +40,31 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.Azure.Management.EventGrid, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+    <Reference Include="Hyak.Common, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
@ahamad-MS there's a few package references being added here that are already being referenced from a separate file (`Common.Dependencies.Test.targets`), so you can remove these and ensure the only package reference added/changed was `Microsoft.Azure.Management.EventGrid`

Reply:
Fixed